### ISSUE TITLE
[TB] Color Theme - WCAG 2.1.1: Ensure elements that have scrollable content are accessible by keyboard (.design-system-editor-right-content)

### ISSUE THREAD

**Issue by kburk1997** [finos/a11y-theme-builder#925]
### Issue
`Ensure elements that have scrollable content are accessible by keyboard` ([scrollable-region-focusable](https://accessibilityinsights.io/info-examples/web/scrollable-region-focusable))
### Target application
[A11y ThemeBuilder](http://localhost:8080/designSystem/Sample)
### Element path
.design-system-editor-right-content
### Snippet
`<div class="design-system-editor-right-content">`
### How to fix
    Fix any of the following:
      Element should have focusable content
      Element should be focusable
### Environment
Microsoft Edge version 125.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.42.0 (axe-core 4.8.4), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
### Screenshot 
![Screenshot of Color Theme page, where there are no focusable items in the highlighted area](https://github.com/finos/a11y-theme-builder/assets/11298269/26fa99fd-4ed6-4f99-91ae-e4bcedfc21e6)

## Proposed Solution
Put a link or other focusable content somewhere in the page
